src/models/subscriptiondata.cpp

```diff
@@ -44,12 +46,53 @@ bool SubscriptionData::fromJson(const QByteArray& json) {
     return false;
   }
 
-  m_planIntervalCount = planData["interval_count"].toInt();
-  logger.debug() << "m_planIntervalCount" << m_planIntervalCount;
-  if (!m_planIntervalCount) {
+  // Billing frequency
+  QString planInterval = planData["interval"].toString();
+  if (planInterval.isEmpty()) {
     return false;
   }
 
+  // Convert `interval` to number of months
+  int planIntervalMonths;
+  if (planInterval == "month") {
+    planIntervalMonths = 1;
+  } else if (planInterval == "year") {
+    planIntervalMonths = 12;
+  } else {
+    logger.error() << "Unexpected interval type:" << planInterval;
+    emit MozillaVPN::instance()->recordGleanEventWithExtraKeys(
+        GleanSample::unhandledSubPlanInterval, {{"interval", planInterval}});
```

Comment:
Should we just leave the second key `interval_count` out or set it to something like `-1` here?